### PR TITLE
Added Unit Tests

### PR DESCRIPTION
**Pull request by ramkumar-sacumen**
Hi Deepak,

As discussed, I wrote unit tests for Calculator and Shapes. Please review the codes

### CODE REVIEW

File: RAM_KUMAR/assignment/Calculator/calculatorunittest.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    self.assertEqual(self.calculator.add(4,7), 11)
+  def test_subtract(self):
+    self.assertEqual(self.calculator.subtract(10,5), 5)
+  def test_multiply(self):
+    self.assertEqual(self.calculator.multiply(3,7), 21)
+  def test_divide(self):
+    self.assertEqual(self.calculator.divide(10,2), 5)
+
+if __name__ == "__main__":
+  unittest.main()
```
Comment (deepakb-sacumen) [inline RAM_KUMAR/assignment/Calculator/calculatorunittest.py:19]: It's good that you tried writing unit tests using unittest framework. But you need to use pytest here. So please change all the unit test as per pytest

File: RAM_KUMAR/assignment/Calculator/calculator.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+        return float(a) + float(b)
+
+    def subtract(self, a , b):
+        return float(a) - float(b)
+
+    def multiply(self, a , b):
+        return float(a) * float(b)
+
+    def divide(self, a , b):
+        return float(a) / float(b)
```
Comment (deepakb-sacumen) [inline RAM_KUMAR/assignment/Calculator/calculator.py:12]: Variable names should not be used as "a" or "b" give some meaningful name. 
Ex-  num1, num2

File: RAM_KUMAR/assignment/Calculator/calculator.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
+class Calculator :
+    def add(self, a, b):
+        return float(a) + float(b)
```
Comment (deepakb-sacumen) [inline RAM_KUMAR/assignment/Calculator/calculator.py:3]: Why do you type casting the variable?? Just remove this

File: RAM_KUMAR/assignment/Inheritance/shapeunittest.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+     
+   def test_rectangle(self):
+     self.assertEqual(Rectangle(10,5).area(), 50) 
+
+   def test_square(self):
+     self.assertEqual(Square(5).area(), 25)
+
+   def test_circle(self):
+     self.assertEqual(Circle(3).area(), 28.274333882308138)
+     self.assertEqual(Circle(3).perimeter(), 18.84955592153876)
```
Comment (deepakb-sacumen) [inline RAM_KUMAR/assignment/Inheritance/shapeunittest.py:22]: Here also same use pytest for writing Unittest

**Comment by ramkumar-sacumen** [comment]
ok Deepak.

*Thanks & Regards,*
*Ramkumar S | QA Lead*
***@***.*** ***@***.***>|M:+919884583690*

**Pull request closed without merging**